test/integration/image-future/svgo-webpack/test/index.test.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-env jest */
+
+import { join } from 'path'
+import {
+  renderViaHTTP,
+  findPort,
+  launchApp,
+  nextBuild,
+  killApp,
+} from 'next-test-utils'
+
+const appDir = join(__dirname, '..')
+let appPort
+let app
+let devOutput
+
+describe('svgo-webpack with Image Component', () => {
```

Comment:
Does this test need to be copied? It doesn't seem to be testing anything specific to `next/future/image`

Reply:
I think its best to assume all tests in `next/future/image` will replace existing tests for `next/image` (in the future 🙂 ) so I would rather have it run twice so we can just delete all of the existing `next/image` tests if/when we get to that point.